docs/components/connectors/custom-built-connectors/connector-templates.md

```diff
@@ -1038,7 +1038,7 @@ You can, for example, allow the user to model and configure the following **REST
       "type": "Hidden",
       "value": "http",
       "binding": {
-        "type": "zeebe:taskDefinition:type"
+        "type": "zeebe:taskDefinition"
       }
```

Comment:
```suggestion
         "type": "zeebe:taskDefinition",
        "property": "type"
      }
```

Reply:
@sbuettner can this be backported 8.6 - 8.8?